test/e2eTests/pages/registerPatientPage.js

```diff
@@ -78,7 +83,8 @@ class RegisterPatientPage extends Page {
   }
 
   selectGender(patient) {
-    this.I.click(patient.person.gender == 'F' ? this.translate('COMMON_MALE') : this.translate('COMMON_MALE'));
+    const gender = patient.person.gender == 'F' ? 'label[for="patientGenderF"' : 'label[for="patientGenderM"';
```

Comment:
how about ``const genderButton = `label[for="patientGender${patient.person.gender}"]` ``?

Reply:
Great!